recipes/gw/meta.yaml

```diff
@@ -31,6 +35,17 @@ requirements:
     - {{ cdt('libxext') }}           # [linux]
     - wget
     - unzip
+    - htslib >=1.12
```

Comment:
most of this should be only in the host section ...

Reply:
Htslib and wget are needed during compilation

Reply:
Move them to `host`! It will still work!

Reply:
Ah ok, thanks 👍